static/app/views/settings/project/server-side-sampling/serverSideSampling.tsx

```diff
@@ -59,8 +59,16 @@ export function ServerSideSampling({project}: Props) {
     }
   }, [currentRules, previousRules]);
 
-  function handleActivateToggle(rule: SamplingRule) {
-    openModal(modalProps => <ActivateModal {...modalProps} rule={rule} />);
+  function handleActivateToggle(ruleId: SamplingRule['id']) {
+    openModal(modalProps => (
+      <ActivateModal
```

Comment:
There is a path where you can activate it here immediately (without the modal).
Let's please also add code for that. We can mock it for now with some alway true boolean constant.

Reply:
That's true, but since the project details endpoint is not yet providing it, I decided to to it in a separate PR and not more our Jira task to done